pkg/ssh/utils.go

```diff
@@ -172,6 +173,7 @@ func ParseScpArgs(options ConnectionScpOptions) (string, string, string, bool, e
 		remotePath = split[1]
 	}
 	remotePath = strings.TrimSuffix(remotePath, "\n")
+	fmt.Println(host, remotePath, localPath, swap)
```

Comment:
Is this intentional or a debug line?

Reply:
thanks, this is a debug I think I will remove